src/test/java/io/confluent/examples/streams/kafka/EmbeddedSingleNodeKafkaCluster.java

```diff
@@ -220,22 +216,24 @@ public void createTopic(final String topic, final int partitions, final int repl
    */
   public void createTopic(final String topic,
                           final int partitions,
-                          final int replication,
-                          final Properties topicConfig) {
+                          final short replication,
+                          final Map<String, String> topicConfig) {
```

Comment:
Why do we want to replace Properties with a generic map here?

Reply:
The new `AdminClient` we are using takes a Map, so I'm just propogating the change out to callers, rather than building the map internally. It's a private API, so it results in simpler code if I just build a map to start with and pass it down, rather than building Properties and then converting it to a map.